src/Compilers/CSharp/Test/Emit3/PartialEventsAndConstructorsTests.cs

```diff
@@ -885,7 +885,7 @@ partial C() { }
             // (3,33): error CS0751: A partial member must be declared within a partial type
```

Comment:
>Assert.True(e.IsSealed); [](http://example.com/codeflow?start=12&length=24)

This is unexpected. Only abstract sealed explicit implementations can be marked as `IsSealed`. This property is not meant to reflect what syntax says, it is meant to to reflect semantics.